src/applications/representative-appoint/components/SelectAccreditedRepresentative.jsx

```diff
@@ -21,23 +22,27 @@ const SelectAccreditedRepresentative = props => {
     goForward,
     goToPath,
   } = props;
-  const [loadingReps, setLoadingReps] = useState(false);
-  const [loadingPOA, setLoadingPOA] = useState(false);
-  const [error, setError] = useState(null);
+
   const representativeResults =
     formData?.['view:representativeSearchResults'] || null;
-
-  const currentSelectedRep = useRef(formData?.['view:selectedRepresentative']);
-
   const query = formData['view:representativeQuery'];
   const invalidQuery = query === undefined || !query.trim();
-
   const noSearchError =
     'Enter the name of the accredited representative or VSO you’d like to appoint';
 
   const noSelectionError =
     'Select the accredited representative or VSO you’d like to appoint below.';
 
+  const [loadingReps, setLoadingReps] = useState(false);
+  const [loadingPOA, setLoadingPOA] = useState(false);
+  const [searchWasPerformed, setSearchWasPerformed] = useState(
+    !!representativeResults?.length,
+  );
+  const [error, setError] = useState(null);
+  const [searchHeaderQuery, setSearchHeaderQuery] = useState('');
```

Comment:
This default value shows when navigating backwards in the flow.


https://github.com/user-attachments/assets/b7b4c54f-a936-48c6-ade0-e452da797759

Reply:
There's an edge case for the review page that may or may not be relevant. If you click edit the rep from review, enter a search without results, click continue, and then click edit again you'll see the new resultless search in the input field but no result description.


https://github.com/user-attachments/assets/a17ed8fb-a641-409e-801d-778d5d392437

Reply:
Edge case now fixed.

Reply:
The query length check introduced a new edge case when you only change the text input without pressing the search button. 

https://github.com/user-attachments/assets/22d72173-5780-452c-acaa-52c0d8ffd249

Reply:
I'm not sure that will ever come up. I can approve the pr as is.